styles.css

```diff
@@ -337,3 +327,1067 @@ body {
   flex-grow: 0;
   margin: 12px 0;
 }
+
+.header-shapes {
+  /* header-shapes mobile */
+
+  position: absolute;
+  height: 600px;
+  left: 0;
+  top: 104px;
+}
+
+.header-background {
+  /* header-background-mobile */
+
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+  visibility: hidden;
+}
+
+.works {
+  /* Works */
+
+  /* Auto layout */
+
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+  position: absolute;
+  width: 327px;
+  height: 2560px;
+  left: 0;
+  top: 718px;
+}
+
+.card-works-1 {
+  /* Card Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 575px;
+  left: 24px;
+  top: 114px;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+
+  /* Neutrals/N40 */
+
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+
+  /* Inside auto layout */
+
+  flex: none;
+  flex-grow: 0;
+  margin: 88px 0;
```

Comment:
- [ ] The margin between each card should be `88px` but due to the way you are specifying it, you tend to have 88px top and bottom. This makes the space between each card twice as wide. Kindly fix this 🔍